src/controllers/IllPersonController.ts

```diff
@@ -61,11 +62,21 @@ export class IllPersonController extends BaseController<FormBody> {
     }
 
     protected prepareSessionModelPriorSave(appeal: Appeal, value: FormBody): Appeal {
+
         if (appeal.reasons?.illness != null) {
+
             appeal.reasons.illness.illPerson = value.illPerson;
+
             if (value.illPerson !== IllPerson.someoneElse) {
+
                 appeal.reasons.illness.otherPerson = undefined;
+
+            } else {
+
```

Comment:
I'm curious about all of the additional whitespace, particularly in this `else` block. Is there a style guide I need to take a look at?

Reply:
I only did that because when debugging I was finding it hard to read. I think its subjective.

Reply:
Fair enough! Just checking :)

Reply:
Im not very consistent, but since we had nested if and else, I thought it might be better